docs/4.1/layout/overview.md

```diff
@@ -70,13 +70,12 @@ Bootstrap primarily uses the following media query ranges—or breakpoints—in
 Since we write our source CSS in Sass, all our media queries are available via Sass mixins:
 
 {% highlight scss %}
-@include media-breakpoint-up(xs) { ... }
```

Comment:
Let's actually swap this with something that explicitly states there's no `xs` query?

```scss
// No media query necessary for `xs` breakpoint as it's effectively `@media (min-width: 0) { ... }
```

Reply:
Good point. People might think 'where is xs'.